stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-gcp-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} VM instances SSL/TLS certificate and key
+
+By default, VM instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want VM instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
+
+.Procedure
+. Locate the correct secret for the {PlatformNameShort} deployment by opening the GCP Secrets Manager console and selecting the secret with the name _<deployment_name>-pulp_cert_.
```

Comment:
. Navigate to the GCP Secrets Manger Console.
. Locate the secret for the {PlatformNameShort deployment, with the name <deployment_name>-pulp-cert. (wrap the whole thing in back ticks and <deployment_name> in underscores.)

Reply:
Fixed